terra2/assetlist.json

```diff
@@ -958,6 +958,50 @@
         }
       ]
     },
+    {
+      "description": "Tether USDt on Terra",
```

Comment:
please relocate this asset to the bottom of the list

Reply:
done